Week1/homework/js-exercises/ex6.js

```diff
@@ -0,0 +1,17 @@
+//Exercise 6: Log an array of animals
+//Declare variable and assign to it an empty array
+const empty = [, , ,];
+// what i think the value of the array is.
+console.log('the arry have empty items, and the type of empty is object');
+//logs the array
+console.log(empty);
+//Create a new variable with an array animal
+const animal = ['cats', 'Dogs', 'Horses'];
+//logs the second array
+console.log(animal[1]);
+//adds another string to the arry
+animal.unshift('Piglet');
```

Comment:
aha, you can also use [push](https://developer.mozilla.org/nl/docs/Web/JavaScript/Reference/Global_Objects/Array/push) (which is more commonly used)

Reply:
changes done, now i understand why to log the same element in the array before and after add new string, it didn't change because i add new string in the end of the array with push,  but when i add it with unshift the add will be first the array and it will change all info of my array.